src/components/ui/card/Card.vue

```diff
@@ -35,6 +45,11 @@ const toggleShowBody = () => {
   showBody.value = !showBody.value;
 };
 
+onMounted(() => {
+  isDropdown.value = props.dropdown;
+  showBody.value = props.dropdownClosed;
```

Comment:
Aqui deveria atribuir negado, a correção ficou errada:
```suggestion
  showBody.value = !props.dropdownClosed;
```